encrypt-app.jsx

```diff
@@ -3,6 +3,8 @@ import React from 'react';
 /* eslint-disable no-unused-vars */
 import { render } from 'react-dom';
 import routes from './encrypt-main.jsx';
+import ga  from 'react-ga';
 import { Router, browserHistory } from 'react-router';
 
+ga.initialize(process.env.GA_TRACKING_ID, {debug: true});
```

Comment:
It should if I set the env before I deploy lol. I should also remove that debug thing